<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChargeInfo</class>
 <widget class="QDialog" name="ChargeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>383</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>762</width>
    <height>383</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Condensed</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_V">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>130</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_A">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_KWh">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>33</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>KWh</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_per">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>240</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>73</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前SOC：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>充电电流：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>充电电量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>消费金额：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>开始时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>充电电压：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>74</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>累计时间：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_begintime">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_totaltimes">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_money">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_current">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_electricity">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SOC">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_right">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>160</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>卡    号：</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cardid">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_voltage">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <zorder>label_9</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_11</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label_4</zorder>
  <zorder>label_V</zorder>
  <zorder>label_A</zorder>
  <zorder>label_KWh</zorder>
  <zorder>label_per</zorder>
  <zorder>lineEdit_begintime</zorder>
  <zorder>lineEdit_totaltimes</zorder>
  <zorder>lineEdit_money</zorder>
  <zorder>lineEdit_current</zorder>
  <zorder>lineEdit_electricity</zorder>
  <zorder>lineEdit_SOC</zorder>
  <zorder>pushButton_right</zorder>
  <zorder>label_2</zorder>
  <zorder>lineEdit_cardid</zorder>
  <zorder>lineEdit_voltage</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
